frontend/packages/data-portal/app/components/Dataset/DatasetMetadataTable.tsx

```diff
@@ -64,7 +48,7 @@ export function DatasetMetadataTable({
 
     !!allFields && {
       label: t('datasetId'),
-      values: [`${dataset.id!}`],
+      values: [`${dataset.id ?? ''}`],
     },
 
     !!allFields && {
```

Comment:
not part of your changes, but why does this need `!!` if it's already a boolean?

Reply:
`getTableData` expects either a data entry or a boolean and since `allFields` can be undefined it complains